kairos-cli/bin/crypto/signer.rs

```diff
@@ -0,0 +1,54 @@
+use std::path::PathBuf;
+
+use super::private_key::{parse_private_key, CasperPrivateKey};
+use super::public_key::CasperPublicKey;
+use crate::crypto::error::CryptoError;
+use casper_types::{bytesrepr::ToBytes, SecretKey};
+use casper_types::{crypto, PublicKey};
```

Comment:
Nit: merge these imports.
RLS has an action for this.

Reply:
Imports formatted: https://github.com/cspr-rad/kairos/pull/17/commits/c314c50b33c09e6f3e9449bc9d80b3d9fca63fca.